### PR TITLE
FacePile: Converts to modular css

### PR DESCRIPTION
**Pull request by aleksandrjPersonal**
#### Pull request checklist

- [ ] Addresses an existing issue: #0000
- [ ] Include a change request file if publishing 
- [ ] New feature, bugfix, or enhancement
  - [ ] Includes tests
- [ ] Documentation update

#### Description of changes

(give an overview)

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/Facepile/Facepile.tsx [hunk 1]
```diff
@@ -142,7 +142,7 @@ export class Facepile extends React.Component<IFacepileProps, {}> {
   private _getAddNewElement(): JSX.Element {
     let { personaSize } = this.props;
     return <button { ...getNativeProps(this.props.addButtonProps, buttonProperties) }
-      className={ css('ms-Persona', PERSONA_SIZE[personaSize], 'ms-Facepile-addButton', 'ms-Facepile-itemButton') }>
+      className={ css('ms-Persona', PERSONA_SIZE[personaSize], 'ms-Facepile-addButton', 'ms-Facepile-itemButton', styles.addButton, styles.itemButton) }>
```
Comment (dzearing) [inline packages/office-ui-fabric-react/src/components/Facepile/Facepile.tsx]: I think this is a bug.. Facepile is creating a button with ms-Persona class, so it's relying on some external css here, and PERSONA_SIZE[personaSize] maps to some css class that won't exist once Persona is converted...

Comment (dzearing) [inline packages/office-ui-fabric-react/src/components/Facepile/Facepile.tsx]: I don't really understand the intent here; why button is used and not Persona itself?

**Comment by micahgodbolt** [comment]
Pulled this down and saw this icon was broken 

image

**Comment by aleksandrjPersonal** [comment]
@micahgodbolt Hmmm, Not seeing the missing padding issue you describe in 'FacePile Add face'

File: packages/office-ui-fabric-react/src/components/Facepile/Facepile.tsx (hunk 1 above)
Comment (aleksandrjPersonal) [inline packages/office-ui-fabric-react/src/components/Facepile/Facepile.tsx]: Maybe because Button has an onClick handler ? Is this blocking this PR?

**Comment by micahgodbolt** [comment]
@aleksandrjPersonal it seems that the ms-Persona--xs styles aren't being applied

Is it working in your branch?

image

**Comment by micahgodbolt** [comment]
Ahh, the issue is actually in persona. I pulled down all of the various PRs. I'll move the concern over to that PR.

**Comment by aleksandrjPersonal** [comment]
good :)

File: packages/office-ui-fabric-react/src/components/Facepile/Facepile.tsx (hunk 1 above)
Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/Facepile/Facepile.tsx]: Hmmm....well i think this is going to cause the bug I saw once combined with persona.

I.e. We can pass classes down to child components, we need to pass data down that triggers the addition of styles.

Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/Facepile/Facepile.tsx]: We might need to merge this and persona in and file a bug against this.

**Comment by micahgodbolt** [comment]
Ok, I don't want to start hacking on a solution to this until this is merged in with persona. So we either need to combine them together before merge, or merge them both in and address the bug. 

The problem I see is that we're applying the persona class (and styles) to a random button in the DOM instead of using an actual persona.

Using a persona is tricky because it has to be inside of a button, which might be ok, but then we need to be able to pass the icon in instead of an image/initials. To do that we need to add an onRender in persona to support passing in an custom icon.

All doable and valuable, but hard to do with both PRs up in the air.

**Pull request closed without merging**